spec/README.md

```diff
@@ -39,30 +39,30 @@ adopted by in-toto in [ITE-5]. It is a [JSON] object with the following fields:
 
 `payloadType` *string, required*
 
->   Always `https://in-toto.io/Statement/v1-json` (for the [Statement] defined
->   below).
+> Identifier for the encoding of the payload. Always
+> `https://in-toto.io/Statement/json` (for the JSON [Statement] defined below).
 
 `payload` *string, required*
 
->   Base64-encoded JSON [Statement].
+> Base64-encoded JSON [Statement].
 
 `signatures` *array of objects, required*
 
->   One or more signatures over `payloadType` and `payload`, as defined in
->   [signing-spec].
+> One or more signatures over `payloadType` and `payload`, as defined in
+> [signing-spec].
 
 ## Statement
 
 ```jsonc
 {
+  "type": "https://in-toto.io/Statement/v1",
```

Comment:
optional: Should we use "_type" to help prevent collisions?

Reply:
In-toto currently uses "_type", so that would also be a collision. Other ideas include "$type", "predicateType", "$predicateType", or variants involving "schema" instead of "type". I don't have much of an opinion here. I went with "type" because it as the simplest. 

Any other thoughts?

Reply:
I think it's quite common to define the format/media type in the outer layer if you want to treat the inner layer as an opaque thing. Whether the inner layer itself is self-describing using some standard or custom method (e.g. JSON-LD) is a separate concern. I would also be worried of injecting properties as some formats are strict about it and are not valid anymore (something that's possible to be expressed in JSON schema for validation for example).
If you want to inject/enforce a certain field then I think you'd have to make stronger statements about what types of formats are supported.

Reply:
Thanks for the feedback, Maik. A few scattered thoughts:

- It seems like a good idea to add `Statement.type` is in case the envelope doesn't authenticate `payloadType`, since not all envelopes do. This seems like a good idea to me.
- I'm less confident about renaming `payloadType` to `payload.type`. That isn't for security - it's just for comprehension. Note that you can always include an opaque value as `payload.value`, which we do for SPDX. But maybe this isn't a good idea.
- Either way, I'm leaning towards differentiating between "type" and "version". If we think about other formats like JPEG, PDF, or JSON Schema, they all have a single Media Type and have a version within the file. Maybe we should use that same concept here. See also #4.

Reply:
Reverted the `predicateType` -> `predicate.type` change, and switched to Media Type for the `payloadType`.